src/controllers/users.controller.js

```diff
@@ -0,0 +1,72 @@
+const { usersService } = require('../services/user.services.js');
+
+const createUser = (req, res) => {
+  const user = req.body;
+
+  if (!user.name) {
+    return res
+      .status(400)
+      .json({ success: false, message: 'Please provide name of user' });
+  }
+
+  const newUser = usersService.createUser(user.name);
+
+  res.status(201).json(newUser);
+};
+
+const getUsers = (req, res) => {
+  const users = usersService.getUsers();
+
+  res.status(200).json(users);
+};
+
+const getUser = (req, res) => {
+  const neededUser = usersService.getUserById(+req.params.id);
+
+  if (!neededUser) {
+    return res.sendStatus(404);
+  }
+
+  res.status(200).json(neededUser);
+};
+
+const deleteUser = (req, res) => {
+  const user = usersService.deleteUserById(+req.params.id);
+
+  if (!user) {
+    return res
+      .status(404)
+      .json({ success: false, message: 'User does not exist' });
+  }
+
+  res.status(204).end();
+};
+
+const updateUser = (req, res) => {
+  const { name } = req.body;
+
+  if (!name) {
+    return res.status(400).json({ message: 'Name is required' });
+  }
+
+  const user = usersService.getUserById(+req.params.id);
+
+  if (!user) {
+    return res.sendStatus(404);
```

Comment:
The error response for a non-existent user should include a descriptive message. Consider using `res.status(404).json({ message: 'User not found' });` to provide more context.